apps/vs-code-designer/src/app/commands/createCodeless/createCodelessSteps/CodelessWorkflowCreateStep.ts

```diff
@@ -45,19 +46,68 @@ export class CodelessWorkflowCreateStep extends WorkflowCreateStepBase<IFunction
   }
 
   public async executeCore(context: IFunctionWizardContext): Promise<string> {
+    // Check if the user chose to initialize a static web app
+    if (context.initializeStaticWebApp) {
+      context.telemetry.properties.initializeStaticWebApp = 'true';
+      vscodeExtension.commands.executeCommand('azureLogicAppsStandard.deploy');
```

Comment:
NIT: Here is just better to call the function if possible, rather than calling the command